src/core/window/interface.js

```diff
@@ -55,9 +55,7 @@ export class WindowInterface {
    * @return {string}
    */
   static getUserLanguage(win) {
-    // The `navigator.userLanguage` is only supported by IE. The standard is
-    // the `navigator.language`.
-    return win.navigator['userLanguage'] || win.navigator.language;
+    return win.navigator.language;
```

Comment:
Is this a safe change? Should it be in this PR?

Reply:
According to the comments it breaks IE, which is ok.

Reply:
My brain says this is fine, my gut says this isn't related to typechecking and isn't a no-op and maybe should be bundled with related IE cleanup/tagged with a tracking issue rather than changing here. I don't feel very strongly though so if you think it's fine then w/e

Reply:
Sure, reverted to be safe. We can remove in a future PR.

Reply:
Moved to: https://github.com/ampproject/amphtml/pull/37207